documentation/smg/11_ISIS_release.md

```diff
@@ -1,79 +1,161 @@
-# ISIS Release Process
+# SMG-11: ISIS Release Process
 
-Horace is deployed separately to a number of systems across ISIS
+Horace is deployed separately to a number of systems across ISIS. These are:
 
-## ISIS Compute
+- IDAaaS: This is currently the normal access point for users of Horace in ISIS.
+- SCARF: This is an alternative HPC environment. In contrast to the deployment on IDAaaS, the current SCARF deployment is experimental, and its parallel capabilities are more limited. 
+Details of this are given below, including contacts for improving the installation.
+- ISISCOMPUTE: This is the base location for code deployed to IDAaaS, and can also be used directly by experienced users only. The section on this below gives reference details
+for use of this system. 
 
-#### Contact
+This document is primarily concerned with the process of upgrading Horace and releasing new Horace versions on IDAaaS. However, details for how we might update SCARF 
+and how experienced users would access Horace through ISISCOMPUTE are also given for completeness.
 
-FBU IT support team: FBUitservicedesk@stfc.ac.uk 
+## IDAaaS
 
+IDAaaS is the cloud computing service "ISIS Data Analysis as a Service" provided by Scientific Computing for ISIS users.
+It is accessed from https://isis.analysis.stfc.ac.uk/, from where you can login to the service. This also gives instructions for new users; similar instructions are also available in the
+registration part of the dialog on the login page. Detailed use of IDAaaS and Matlab on IDAaaS is outside the scope of this document.
 
-#### Deploy Location
-The physical location of the code on ISISCOMPUTE file system is:
-
-`/home/isis_direct_soft/Horace_git`
-`/home/isis_direct_soft/Horace_bugfix`
-`/home/isis_direct_soft/Herbert_git`
-`/home/isis_direct_soft/Herbert_bugfix`
+#### Contact
 
-By agreement, this location is accessed by code users through symbolic links, created in `/usr/local/mprogs` folder.
+IDAaaS support team : support@analysis.stfc.ac.uk
+
+#### Main Deploy Location
+
+Various versions and components of Horace are deployed in a number of local repositories under the path `/usr/local/mprogs`.
+
+The physical disk location of these Horace repositories is the same as for ISISCOMPUTE system; the description for that system below gives more details.
+As this may change subject to development of the underlying IDAaaS system structure, the path `usr/local/mprogs` provides symbolic links to the repositories' current locations.
```

Comment:
`/usr/local/mprogs` -- first dash is important

Reply:
thanks